src/objective-c/tests/Tests.xcodeproj/project.pbxproj

```diff
@@ -367,12 +367,15 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 5E8A5DAE1D3840B4000F8BC4 /* Build configuration list for PBXNativeTarget "CoreCronetEnd2EndTests" */;
 			buildPhases = (
+				19087EB522BF2CB904F217C6 /* 📦 Check Pods Manifest.lock */,
```

Comment:
You shouldn't run Cocoapods 1.0.0 or it'll add these redundant phases to your project file :)

After 1.0.0 the 📦 was changed to `[CP]` (like in the line just below this).